osu.Game/IO/Archives/LegacyFilesystemReader.cs

```diff
@@ -15,7 +14,8 @@ public class LegacyFilesystemReader : ArchiveReader
     {
         private readonly string path;
 
-        public LegacyFilesystemReader(string path) : base(Path.GetFileName(path))
+        public LegacyFilesystemReader(string path)
+            : base(Path.GetFileName(path))
         {
             this.path = path;
```

Comment:
We should ensure that this path is the same that we'd expect from the `Directory.GetFiles()` call below.

`path = Path.GetFullPath()` should fulfil this.